src/Domain/Model/Euro.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Domain\Model;
+
+use InvalidArgumentException;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+final class Euro {
+
+	private static $DECIMAL_COUNT = 2;
+	private static $CENTS_PER_EURO = 100;
```

Comment:
Yeah - they can be private. Once we switch to PHP 7.1, we can have private consts. Not that it matters much anyway